overlay/disquick/disctl/src/main.rs

```diff
@@ -21,7 +21,9 @@ View and modify local service state.
 Options:
   -e --stop         End the service
   -l --start        Launch the service
+  -r --restart      Restart the service
   -j --journal      Show service journal
+  -n --reverse      Show service journal with new entries first
```

Comment:
What's the usability of this vs. `journalctl -e`? You can actually combine `-e` and `-f`, which is kinda nice.